holoviews/operation/datashader.py

```diff
@@ -307,7 +334,12 @@ def _process(self, element, key=None):
 
         with warnings.catch_warnings():
             warnings.filterwarnings('ignore', r'invalid value encountered in true_divide')
-            img = tf.shade(array, **shade_opts)
+            if np.isnan(array.data).all():
```

Comment:
I worried about that too, but 100ms for a 10000x10000 array (which is considerably larger than we'll ever use), is okay.